vertx-sql-client/src/main/java/io/vertx/sqlclient/impl/Connection.java

```diff
@@ -23,10 +23,14 @@
 
 public interface Connection extends CommandScheduler  {
 
+  default boolean isLazyException(Throwable t) {
```

Comment:
what does lazy exception mean?

Reply:
it means it checks whether the exception is a lazy statement, I'll try finding a better name, for now this PR is not polished at all but it provides most of what we want to do.

Reply:
something like isIncompletePreparedStatementException(T) would be better likely